breadbox/tests/api/test_compute.py

```diff
@@ -355,8 +353,8 @@ def test_run_custom_analysis_pearson_with_feature_ids_and_query_values(
 ):
     """
     The legacy portal makes custom analysis requests to breadbox with a slightly different format.
-    1. It uses feature ids instead of node ids (with the flag use_feature_ids=True)
-    2. It sometimes passes in pre-loaded query values instead of using the query_node_id parameter
+    Instead of using the uery_feature_id + query_dataset_id params, it sometimes passes in 
+    pre-loaded query values. 
```

Comment:
I'm a little confused about what it means to pass in pre-loaded query values.

Reply:
Good point! I think what this is trying to describe is the way that the feature/"query vector" data is sometimes loaded in the `portal-backend` and _then_ passed to breadbox (instead of just loading the data from within breadbox). This is one of many things that could probably be simplified once we have the fronted talking directly to breadbox. I'll update that comment to make it more clear. 👍

Reply:
I personally find this architecture really confusing, but it's been a necessary evil during the breadbox migration. I'm hoping that we can remove a lot of this complexity soon!

Reply:
Something I think about sometimes: The **entire** portal backend worker can go away once all data is in breadbox.
No more confusing shared code between the two services. No more `./flask run_worker`

That's a lot of extra moving parts that we'll be able to get rid of as soon as we can rely on breadbox for all dat.